src/test/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/ConjugatedPiSystemFragmenterTest.java

```diff
@@ -0,0 +1,164 @@
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package de.unijena.cheminf.mortar.model.fragmentation.algorithm;
+
+import de.unijena.cheminf.mortar.model.util.ChemUtil;
+
+import javafx.beans.property.Property;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openscience.cdk.interfaces.IAtomContainer;
+import org.openscience.cdk.silent.SilentChemObjectBuilder;
+import org.openscience.cdk.smiles.SmilesParser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Class to test the correct working of
+ * {@link de.unijena.cheminf.mortar.model.fragmentation.algorithm.ConjugatedPiSystemFragmenter}.
+ *
+ * @author Maximilian Rottmann
+ * @version 1.1.1.0
+ */
+public class ConjugatedPiSystemFragmenterTest {
+
+    /**
+     * Constructor that sets the default locale to british english, which is needed for correct functioning of the
+     * fragmenter as the settings tooltips are imported from the message.properties file.
+     */
+    public ConjugatedPiSystemFragmenterTest() {
+        Locale.setDefault(new Locale("en", "GB"));
+    }
+
+    /**
+     * Tests correct instantiation and basic settings retrieval.
+     *
+     * @throws Exception if anything goes wrong
+     */
+    @Test
+    public void settingsTest() throws Exception {
+        ConjugatedPiSystemFragmenter tmpFragmenter = new ConjugatedPiSystemFragmenter();
+        List<String> tmpCheckList = new ArrayList<>();
+        List<String> tmpExpectList = new ArrayList<>();
+        tmpExpectList.add("Fragment saturation setting");
+        for (Property tmpSetting: tmpFragmenter.settingsProperties()) {
+            tmpCheckList.add(tmpSetting.getName());
+        }
+        Assertions.assertTrue(this.compareListsIgnoringOrder((ArrayList) tmpExpectList, (ArrayList) tmpCheckList));
+    }
+
+    /**
+     *  Method to test a default conjugated pi system fragmentation on the natural product CNP0421388
+     *  from the Coconut Database (@see <a href="https://coconut.naturalproducts.net/compound/coconut_id/CNP0421388">...</a>).
+     *
+     * @throws Exception if anything goes wrong
+     */
+    @Test
+    public void defaultFragmentationTest() throws Exception {
```

Comment:
general comment on the test class: I think this is enough. Since this is a CDK utility we are simply using out of the box, we do not need to test it extensively (again).